### PR TITLE
fix: include Stencil in build

### PR DESCRIPTION
**Pull request by jgroth**
fix Lundalogik/crm-feature#2898

`getElement` from Stencil is used by the decorators and is needed in the build

### CODE REVIEW

File: packages/lime-web-components/package.json [hunk 1]
```diff
@@ -40,5 +40,11 @@
     "@limetech/lime-web-components-decorators": "^4.51.1",
     "@limetech/lime-web-components-interfaces": "^4.51.1",
     "@limetech/lime-web-components-testing": "^4.51.1"
+  },
+  "peerDependencies": {
+    "@stencil/core": "^2.17.1",
+    "@types/jest": "^27.5.0",
+    "jest": "^27.5.1",
+    "jest-cli": "^27.5.1"
```
Comment (vicgeralds) [inline packages/lime-web-components/package.json:48]: To me it doesn't make any sense to have jest as a peer dependency. Why?

**Review by vicgeralds — CHANGES_REQUESTED** [review]
Good to have stencil as a peer dependency. 👍
Not sure why we need this when we didn't before 😄 

The package-lock files are not in sync (means `npm ci` fails) and seem generated by lerna (reformatted with different indentation than when you do `npm install`).

File: packages/lime-web-components/package.json (hunk 1 above)
Comment (Befkadu1) [reply packages/lime-web-components/package.json:47]: What is the problem having jest in `devDependencies`?

Comment (vicgeralds) [reply packages/lime-web-components/package.json:47]: @Befkadu1 no one said there was a problem having jest in devDependencies. 🙁

Comment (vicgeralds) [reply packages/lime-web-components/package.json:47]: He moved them from `peerDependencies`, I can see the comment is confusing without the original context

**Review by vicgeralds — DISMISSED** [review]
I updated the package-lock.json files 🙂

**Comment by jgroth** [comment]
> I updated the package-lock.json files slightly_smiling_face

Great! :slightly_smiling_face:

**Pull request merged**